lib/printers/pretty.js

```diff
@@ -12,7 +12,7 @@ class PrettyPrinter {
     for (const filePath of Object.keys(results.files)) {
       let fileResults = results.files[filePath];
       let messages = this.options.quiet
-        ? fileResults.messages.filter((r) => r.severity !== Linter.WARNING_SEVERITY)
+        ? fileResults.messages.filter((r) => r.severity === Linter.ERROR_SEVERITY)
```

Comment:
Lets tweak this to be `> Linter.WARNING_SEVERITY`?

Reply:
Done.